sdk/lib/crt/mbstring/mbsstr.c

```diff
@@ -8,16 +8,20 @@ unsigned char *_mbsstr(const unsigned char *src1,const unsigned char *src2)
 {
   size_t len;
 
-  if (src2 ==NULL || *src2 == 0)
+  if (_mbslen(src2) > _mbslen(src1))
```

Comment:
`_mbslen()` does not seem to support `NULL`.
Current function does not seem to handle `NULL` `src1` either...